### PR TITLE
TSK-1701: added integration test for task routing via REST API.

### PR DESCRIPTION
**Pull request by holgerhagen**
---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java [hunk 1]
```diff
@@ -668,6 +669,31 @@ void testCreateWithPlannedAndDueDate() {
     assertThatThrownBy(httpCall).isInstanceOf(HttpStatusCodeException.class);
   }
 
+  @Test
+  void should_RouteCreatedTask_When_CreatingTaskWithoutWorkbasketInformation() {
+    TaskRepresentationModel taskRepresentationModel = getTaskResourceSample();
+    taskRepresentationModel.setWorkbasketSummary(null);
+    ;
```
Comment (gitgoodjhe) [inline rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java:676]: Can we remove this? Looks weird :)

**Review by gitgoodjhe — CHANGES_REQUESTED** [review]

**Review by gitgoodjhe — APPROVED** [review]
